src/mon/OSDMonitor.cc

```diff
@@ -385,12 +385,17 @@ bool OSDMonitor::thrash()
 
   // generate some pg_temp entries.
   // let's assume the ceph::unordered_map iterates in a random-ish order.
-  int n = rand() % mon->pgmon()->pg_map.pg_stat.size();
+  int pg_num = mon->pgmon()->pg_map.pg_stat.size();
```

Comment:
might want to check for 0 at https://github.com/ceph/ceph/pull/10739/commits/80051092996a5427c80840060003512362e2f578#diff-0a5db46a44ae9900e226289a810f10e8R349 also.